tests/post_training/data/ptq_reference_data.yaml

```diff
@@ -18,7 +18,7 @@ hf/hf-internal-testing/tiny-random-GPTNeoXForCausalLM_statefull_backend_OPTIMUM:
   metric_value: null
 hf/hf-internal-testing/tiny-random-GPTNeoXForCausalLM_stateless_backend_OPTIMUM:
   metric_value: null
-  xfail_reason: "Issue-161969"
+  exception_xfail_reason: "Issue-161969"
```

Comment:
For exception need to know exception type and msg to set xfail only for expected error
```
exception_xfail_reason:
  msg: "Issie-123"
  class: RuntimeError
  errmsg: "some error"
```